src/lib.rs

```diff
@@ -97,6 +97,18 @@ impl MetalIrConverter {
             funcs: self.funcs.clone(),
         })
     }
+
+    #[doc(alias = "IRRayTracingPipelineConfigurationCreate")]
+    pub fn create_ray_tracing_pipeline_configuration(&self) -> IRRayTracingPipelineConfiguration {
```

Comment:
This new object seems to supersede `IRCompilerSetRayTracingPipelineArguments`, all the setters together contain the fields that this function has (plus more iirc).